frontend/src/utils/signHelpers/FormikSchemas/signIN_Schema.tsx

```diff
@@ -0,0 +1,15 @@
+import { string, object } from 'yup';
+import {
+  login,
+  signInMinHeight,
+  signInMaxHeight,
+  requiredLogin,
+  requiredPass,
+} from '../signReceiver';
+
+const signInSchema = object().shape({
```

Comment:
Please move all the validations to the separate folder src/validations (signInValidationSchema)

Reply:
Replaced in scr and renamed